src/NLog.StructuredLogging.Json.Tests/LoggingException.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+
+namespace NLog.StructuredLogging.Json.Tests
+{
+    [Serializable]
```

Comment:
I think this needs to be conditionally compiled like the stack trace access as I think serialization was removed in .NET Core, though I may have confused it with binary serialization, rather than just the attribute.